README.rst

```diff
@@ -9,9 +9,9 @@ Installation
 
 ::
 
-    pip install git+https://github.com/lpm0073/tutor-contrib-credentials
+    pip install git+https://github.com/overhangio/tutor-contrib-credentials.git
```

Comment:
Can you please replace all instances of "contrib-credentials" by "credentials"? (including in setup.py)